kompose.spec

```diff
@@ -967,6 +973,18 @@ export LDFLAGS=%{ldflags}
 %doc RELEASE.md CHANGELOG.md README.md CONTRIBUTING.md ROADMAP.md code-of-conduct.md
 %{_bindir}/kompose
 
+%post
+if [ -z ${BASH+x} ];
+then
+  %{_bindir}/kompose completion zsh > %{_datadir}/zsh/site-functions/kompose;
+else
+  %{_bindir}/kompose completion bash > %{_datadir}/bash-completion/completions/kompose;
+fi
```

Comment:
so my thoughts are we should create these files earlier in the process and include them in the rpm. so basically we compile kompose and the use the binary to create these files. then include them in %files. Also, lets create both zsh and bash completion files to be delivered by the rpm.